cmd/sonicd/metrics/disksize.go

```diff
@@ -14,17 +15,20 @@ var once sync.Once
 
 func SetDataDir(datadir string) {
 	once.Do(func() {
-		go measureDbDir("db_size", datadir)
-		go measureDbDir("statedb/disksize", filepath.Join(datadir, "carmen"))
+		go measureDbDir(context.Background(), "db_size", datadir)
+		go measureDbDir(context.Background(), "statedb/disksize", filepath.Join(datadir, "carmen"))
 	})
 }
 
-func measureDbDir(name, datadir string) {
+func measureDbDir(ctx context.Context, name, datadir string) {
 	var (
-		gauge = metrics.GetOrRegisterGauge(name, nil)
+		gauge  = metrics.GetOrRegisterGauge(name, nil)
```

Comment:
nit: probably a formatting issue, like the one this started with.

Reply:
this is the one that started this.